src/DataCollector/DoctrineDataCollector.php

```diff
@@ -165,10 +174,14 @@ public function collect(Request $request, Response $response, Throwable|null $ex
             }
         }
 
-        $this->data['entities'] = $entities;
-        $this->data['errors']   = $errors;
-        $this->data['caches']   = $caches;
-        $this->groupedQueries   = null;
+        // Sort entities by count (in descending order)
+        arsort($entityCounts);
```

Comment:
this does not make sense as this variable does not contain a map of entities to count anymore, but a map of manager names to such map. You need to sort `$entityCounts[$name]` after creating it for each entity manager instead (after line 125).